great-idea/css/index.css

```diff
@@ -63,28 +63,102 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container {
+  width: 880px;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  flex-direction: column;
```

Comment:
Nice - to help you out later on there is this cool thing you can do with flex-wrap and direction and combine it into a property called flex-flow and can set it like this:
```css
flex-flow: column wrap;
```